agimus_controller/tests/test_ocp_param_base.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import numpy as np
+
+<<<<<<< Updated upstream
+from agimus_controller.agimus_controller.ocp_param_base import OCPParamsCrocoBase
+from agimus_controller.agimus_controller.trajectory import (
+    WeightedTrajectoryPoint,
+    TrajectoryPoint,
+    TrajectoryPointWeights,
+)
+
+=======
+from agimus_controller.ocp_param_base import OCPParamsCrocoBase
+from agimus_controller.trajectory import WeightedTrajectoryPoint, TrajectoryPoint, TrajectoryPointWeights
+>>>>>>> Stashed changes
```

Comment:
@ArthurH91... Why did you commit merge conflict?  Can you do `pre-commit install`?

Reply:
I do not accept in this case answer that you are just chill guy